src/containers/treePage/ducks/selectors.ts

```diff
@@ -40,6 +42,8 @@ export const mapStewardshipToTreeCare = (
       if (item.watered) activityStrings.push('watered');
       if (item.weeded) activityStrings.push('weeded');
       return {
+        activityId,
```

Comment:
🔧 The [build](https://github.com/Code-4-Community/speak-for-the-trees-frontend/actions/runs/3976899519/jobs/6817668046) is failing because the `TreeCare` interface was changed. We can check this locally by running `npm run check` in the terminal. And this can be fixed by changing lines 54 through 72 in the relevant test file (`src\containers\treePage\test\selectors.test.ts`) to the following (just adding the correct `activityId` and `userId` to the expected output):

```
    it('returns correctly formatted strings with different stewardship combinations', () => {
      const expectedTreeCare: TreeCare[] = [
        {
          activityId: 0,
          userId: 1,
          day: '1st',
          month: 'Jan',
          year: 2021,
          message: 'Was mulched and watered and weeded.',
        },
        {
          activityId: 1,
          userId: 1,
          day: '23rd',
          month: 'Feb',
          year: 2021,
          message: 'Was cleared of waste.',
        },
      ];
```